src/pudl/output/pudltabl.py

```diff
@@ -210,6 +176,8 @@ def _register_output_methods(self: Self):
             "out_eia923__AGG_fuel_receipts_costs": "frc_eia923",
             "out_eia923__AGG_generation": "gen_original_eia923",
             "out_eia923__AGG_generation_fuel_combined": "gf_eia923",
+            # EPA CAMD to EIA crosswalk
+            "core_epa__assn_eia_epacamd": "epacamd_eia",
```

Comment:
Added this here to remove the last ref to `self.pudl_engine` and since it didn't actually seem to be a special case (see below).